docs/topics/api/addons.rst

```diff
@@ -705,7 +709,10 @@ This endpoint allows the metadata for an existing version to be edited.
     .. note::
         This API requires :doc:`authentication <auth>`, and for the user to be an author of the add-on.
 
-.. http:patch:: /api/v5/addons/addon/(int:addon_id|string:addon_slug|string:addon_guid)/versions/(int:id)/
+    .. note::
+        This API accepts both version ids and version numbers in the URL. If the version number passed does not contain any dot characters (``.``) it would be considered an ``id``. To avoid this and force a lookup by version number, add a ``v`` prefix to it.
```

Comment:
Nit: in Firefox, we call it the version "string", not "number". https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/version

Reply:
Ah... let's try to be consistent... I can see a couple places in our docs where we call that `version number`, or even `version number string` that I can fix... 

OTOH that page you linked to also has a header titled `Access the version number in code` ...

Reply:
> OTOH that page you linked to also has a header titled `Access the version number in code` ...

😭

Reply:
https://github.com/mozilla/addons/issues/1465